client/one-time-payment/react/README.md

```diff
@@ -0,0 +1,36 @@
+# One-Time Payments React Sample Integration
+
+This React sample integration uses Vite to spin up an application with a React-Typescript template. Vite is used for the local web server and provides the following functionality:
+
+1. Runs the Vite development server on port 3000 where the React App is accessible.
+2. Proxies requests that start with /paypal-api to the backend server running on port 8080.
+
+## How to Run Locally
+
+Run the React UI application and the Node.js backend API server concurrently.
+
+```bash
+v6-web-sdk-sample-integration/server/node
+npm install
+npm run start
+
+v6-web-sdk-sample-integration/client/one-time-payment/react
+npm install
+npm run dev
```

Comment:
Can we have `npm start` be the command we teach for local dev? Thats the convention we have gone with so far. And for the node js server code that uses typescript, we added `npm run start:production` to differentiate for what would run the built js for prod.

```suggestion
npm start
```